src/aws-cpp-sdk-core/include/aws/core/client/AWSClient.h

```diff
@@ -191,6 +191,7 @@ namespace Aws
             virtual void SetServiceClientName(const Aws::String& name);
 
             void AppendToUserAgent(const Aws::String& valueToAppend);
+
```

Comment:
looks like when you removed the function from the diff you left a white space change behind